src/components/organisms/AssetActions/Pool/Remove.tsx

```diff
@@ -21,6 +21,7 @@ import UserLiquidity from '../../../atoms/UserLiquidity'
 import InputElement from '../../../atoms/Input/InputElement'
 import { useOcean } from '../../../../providers/Ocean'
 import { useWeb3 } from '../../../../providers/Web3'
+import BigNumber from 'bignumber.js'
```

Comment:
not sure if we should prefer either package or if both can do the same but just when searching through code noticing that we use decimal.js in way more places than big number.js. At same time we specifically install decimal.js in package.json, but do not do that for big number.js, which gets installed from web3.js

Reply:
I'll test with decimal.js

> ideally also replace the bignumber.js usage in here and in Consume.tsx

What about `subgraph.ts` - we were using `bignumber.js` only there and in `Consume.tsx`

Reply:
oh right, that's in `main`, I was searching in this branch. So also suggest rebasing or merging `main` into here.

Then looks like ocean.js also uses decimal.js for calculations within `OceanPool.ts` over there, so another reason to use same library over here

Reply:
While i agree we should stick to one library, we should do that in another pr. We need this to fix some important bugs

Reply:
yes, but here we are introducing a new usage so just want to prevent that. Agree this here should be isolated as bug fix and we don't need to change in other places, but should use same library ocean.js is using

Reply:
I agree with Mihai, it'll be better to fix the current issue (including changing `bignumber.js` to `decimal.js` in `Remove.tsx`) and I'll create another PR for switching to `decimal.js` in the other files. Btw I updated the code to use `decimal` + removed part of the things we don't need anymore (`calculateAmountOfOceansRemoved` function).